projects/rawkode-academy/web/src/components/partner/pricing/cards.vue

```diff
@@ -9,7 +9,7 @@ interface PricingPlan {
 const pricingPlans: PricingPlan[] = [
 	{
 		title: "Nip",
-		price: 12000,
+		price: 12,000,
```

Comment:
all the commas were just to make it visually easier to grok immediately what the cost was, but if leaving them out was intentional, I can easily rebase these out (or you can, obvi)